rails-app.sh

```diff
@@ -56,7 +56,7 @@ if [[ ${GIT_BRANCH} != "origin/master" ]]; then
     --cached \
     --format html --out rubocop-${GIT_COMMIT}.html \
     --format clang \
-    app test lib
+    app config lib test spec
```

Comment:
Yes, I think it would. I thought the list was originally there to prevent checks on some parts that we didn't think this would help (e.g. cucumber steps etc).

But I'm happy to just check everything! I will remove that.